libs/components/theme/src/lib/styles/_modals.scss

```diff
@@ -2,3 +2,8 @@
   height: 100%;
   overflow: hidden;
 }
+
+.footer-error-overflow {
```

Comment:
What would you think about putting this in the `_theme` file and then calling it `sky-errors-overflow`. 

I'm not in love with putting something that is used outside of modal in the `_modals` file and would prefer something that starts with `sky` like everything else.

Reply:
i like that!